doc/content/reference/telemetry/cli.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "CLI Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the CLI's user"
+aliases: "/telemetry/cli"
+---
+
+{{% tts %}}\'s [CLI]({{<ref "/reference/cli">}}) collects **completely anonymous** telemetry data, which is used to help us understand how users interact with our product. For example, the current data helps to have a better insight on what type of machines the CLI is more commonly used. Telemetry collection is enabled by default and does not collect personally identifiable information. The following section section explains what information is collected and how it is anonymised.
+
+## Identification
```

Comment:
Identification of?

Reply:
Maybe `User's Identification` would be better?

Reply:
yes, that would be better